src/actions/getUsersAction.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+import { GET_USERS } from "../types";
+
+export let getUsers = users => ({ type: GET_USERS, payload: users });
+
+export let fetchUsers = () => dispatch => {
+  return axios
+    .get("https://develop.websiteone.agileventures.org/api/v1/users")
```

Comment:
Shouldn't this URL come from somewhere else centralized?

Reply:
we did have it set up in the package.json as a proxy, but then it broke the implementation.. I think that was before we added the `rack-cors` gem though... so I'll try extracting it again